web/styles/inbox.css

```diff
@@ -351,7 +356,7 @@
             }
 
             .unread_mention_info {
-                grid-area: unread_mention_info;
```

Comment:
This grid area should be defined for the `.mention-and-unread-count-container`

Reply:
Done!